### ISSUE TITLE
Wrong human population size?

### ISSUE THREAD

**Issue by regevs** [popsim-consortium/stdpopsim#1471]
The default human (HomSap) population size is 10,000 (https://popsim-consortium.github.io/stdpopsim-docs/stable/cli_arguments.html#HomSap).

However, observed heterozygosity levels in humans are 0.0007-0.001, which translates to a diploid population size of ~13500-19300, assuming a mutation rate of 1.29e-8 (the given default). 

I suggest the default to be changed to a reasonable value within this range, e.g. 15,000.

**Comment by regevs** [comment]
Do you require a reference for the population size, or would a reference for the heterozygosity suffice?

**Comment by petrelharp** [comment]
We do require a reference, but a reference for heterozygosity also works. However, changing existing values is not something we've ventured in to, and we'll be pretty hesitant to do it (because of backwards compatibility) unless there's an actual error. I think you're arguing that the number 10,000 doesn't provide the most accurate level of heterozygosity, but it's not wrong (at least, it matches the cited paper) - other than to the extent that all models are wrong.

**Comment by regevs** [comment]
I would say that it's not just that the number 10,000 doesn't provide the most accurate estimate, but that it's significantly off - by 50%-100%. This has significant implications on simulating from the model, like the observed number of mutations, the distribution of coalescence times, etc.; indeed this is how I stumbled upon this inaccurate estimate.

With respect to (Takahata 1993), this was a long time ago, and definitely before the age of abundant whole genome sequencing. Specifically, their Ne=10,000 estimate seems to be based on:

(i) An estimate of heterozygosity of 0.00076 from Li and Sadler 1991, from 75,000bps
(ii) A mutation rate estimate of 1.8-2.4e-8 per bp per generation (see their table 1)

Their heterozygosity estimate is OK, but our current estimate of mutation rate is ~1.25e-8 (Scally and Durbin, 2012); stdpopsim itself uses 1.29e-8. So, this is around 50% off, and so their Ne estimate is outdated if not wrong, I believe.

Figure 1b, in (Mallick et al. Nature, 2016) "The Simons Genome Diversity Project: 300 genomes from 142 diverse populations", shows modern estimates of heterozygosity values for a possible reference.

**Comment by petrelharp** [comment]
Ah, that's a good reference. Here's the figure in question, which shows heterozygosity differing by a factor of ~2 between different groups of people:
![Screenshot from 2023-04-28 05-21-16](https://user-images.githubusercontent.com/1046249/235146064-22ce183d-94d0-4608-a226-668888d50421.png)
I think that for particular use cases one value of Ne would definitely be better than others, if one is using a single value of Ne, but what exactly that value is will depend on the use case. (e.g., what the composition of the people you're studying is).  I think there is a good case that the mutation rate is wrong, since that's by my understanding unlikely to vary much on average between different regions; however, before we change that we'd want to have a serious think about what the implications are of changing something like that for people's workflows. If you feel like you can do that (and that it's a good idea), go for it.

As a more general point, I think that someone using a single-Ne demographic history should only expect to get a pretty rough approximation to genetic diversity, especially as the latter can depend substantially on where they get their samples from.

**Comment by regevs** [comment]
I agree with both statements that a single Ne analysis should only give a rough approximation, and that the usefulness of this single value depends on the samples. However, to the extent that a single number is provided at all in stdpopsim, it still seems to me that it should be in the relevant range of 15,000-20,000 and not a 50% approximation.

That said, I agree the point about backward compatibility is important. How would one make a decision about the implications?

Maybe as an intermediate solution, there could be some additional note in the docs noting this issue.

**Comment by petrelharp** [comment]
I'm closing this; feel free to re-open if you feel like it's not addressed.

**Comment by regevs** [comment]
You are right, the lower range does include 10,000. Seems OK to close - hopefully this thread will be useful if someone stumbles upon the same issue.
